index.html

```diff
@@ -8,12 +8,145 @@
     <link rel="stylesheet" href="css/normalize.css">
     <link rel="stylesheet" href="css/style.css">
     <link rel="shortcut icon" type="image/x-icon"  href="favicon.ico">
+    <script src="https://kit.fontawesome.com/a6d80728cd.js" crossorigin="anonymous"></script>
 </head>
 <body>
+      
+
+
+    <header>
```

Comment:
The code is not indented consistently which makes it harder to read.
You can use _Prettier_ which automatically and indents code for you.
You can read about our code style guide here https://syllabus.codeyourfuture.io/guides/code-style-guide